sandbox/grommet-app/src/pages/sticker-sheet/content/Inputs/Selects.jsx

```diff
@@ -1,16 +1,57 @@
-import { Select } from 'grommet';
+import styled from 'styled-components';
+import { Box, Select, Text } from 'grommet';
 import ContentPane from '../../../../components/ContentPane';
 import { Compare } from '../../components';
+import { optionBefore } from '../../../../themes/theme';
+import { StyleInProgress } from '../../components';
 
+const CustomOption = styled(Box)`
```

Comment:
FYI -- here is my proposal for how grommet teams can achieve the "bookend" easily. I'm thinking we should either::
- Have a recipe on our site for it
- Export it with grommet-theme-hpe